src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -399,6 +399,10 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
                     'version' => $config['filesystem']['s3']['version'],
                 ];
 
+                if (isset($config['filesystem']['s3']['endpoint'])) {
```

Comment:
Please add an other test that does not modify the endpoint and skips this line

Reply:
Isn't this one already skipping this line? https://github.com/sonata-project/SonataMediaBundle/pull/1774/files#diff-217ffd8f1233ee89cb0db792bd11981dL220-L236

Reply:
Only half of tests go inside the if

![Captura de pantalla 2020-07-31 a las 14 01 30](https://user-images.githubusercontent.com/1137485/89033104-62f2de80-d336-11ea-809a-af1e694fdded.png)